cluster-tests/tests/directoryservices/test_001_activedirectory.py

```diff
@@ -286,29 +286,24 @@ def test_010_xattrs_writable_via_smb(request):
 
 def test_030_test_share_failover(request):
     depends(request, ['DS_SMB_SHARE_IS_WRITABLE'])
+    # temporarily disabled while refactoring test code
```

Comment:
What's going on here?

Reply:
Initially refactored to use different network for cluster backend traffic and didn't fully back-out changes when I realized it might cause problems for releng.